Tensile/SolutionStructs.py

```diff
@@ -2485,9 +2485,17 @@ def assignDerivedParameters(state):
       # warn("PersistentKernelAlongBatch not support GSU on HIP, forcing PersistentKernelAlongBatch = False")
       state["PersistentKernelAlongBatch"] = False
 
-    if state["PrefetchAcrossPersistent"] and (state["PersistentKernel"] == 0):
-      # warn("PrefetchAcrossPersistent requires PersistentKernel != 0, forcing PrefetchAcrossPersistent = False")
-      state["PrefetchAcrossPersistent"] = False
+    if state["PrefetchAcrossPersistent"]:
+      if state["KernelLanguage"] == "Source" or \
+         state["PersistentKernel"] == 0 or \
+         state["PrefetchGlobalRead"] == 0 or \
+         state["SuppressNoLoadLoop"]:
+        #warn("PAP requires Assembly, PK != 0, PGR != 0, SuppressNoLoadLoop = True, forcing PAP = False")
+        state["PrefetchAcrossPersistent"] = False
+
+    # TODO- fix this, avoid the bug for now
+    if state["PrefetchAcrossPersistent"] and state["StaggerU"] == 0:
+        state["PrefetchAcrossPersistent"] = False
```

Comment:
warning here too could help as reminder to implement feature later if needed

Reply:
Same as above